### PR TITLE
[#11399] clarify sync behaviour

### LINKED ISSUE #11399: Improve documentation for SORMAS 2 SORMAS setup [5]
### Problem Description

To be refined: Document what current gaps are and what needs to be improved.

### Proposed Solution
Description of S2S as markdown in root docs folder

- [x] properties
- [x] components
- [x] certifactes and key material
- [x] deactivating S2S
- [ ] remove outdated wiki content and add reference to new docs into wiki where needed
  https://github.com/hzi-braunschweig/SORMAS-Project/wiki/Server-Configuration-Options#sormas2sormas-interface
  https://github.com/hzi-braunschweig/SORMAS-Project/wiki/Creating-a-SORMAS2SORMAS-Certificate

### Additional Information

Once we have created/update the documentation, we should present this, followed-up by a lessons learned discussion.

### PR DESCRIPTION
**Pull request by JonasCir**
Fixes #11399 

@leventegal-she : Can you add one quick description of the ignore property to this PR?

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11696

File: docs/SORMAS2SORMAS.md [hunk 1]
```diff
@@ -46,30 +46,30 @@ how to generate a truststore.
...
+| `sormas2sormas.etcd.keyPrefix`                   | The etcd key space prefix which is used to store s2s related information.                                                                              |
+| `sormas2sormas.ignoreProperty.additionalDetails` | Business logic switch to control if the value is ignored when shared/overwritten trough S2S.                                                           |
+| `sormas2sormas.ignoreProperty.externalId`        | Business logic switch to control if the value is ignored when shared/overwritten trough S2S.                                                           |
+| `sormas2sormas.ignoreProperty.externalToken`     | Business logic switch to control if the value is ignored when shared/overwritten trough S2S.                                                           |
+| `sormas2sormas.ignoreProperty.internalToken`     | Business logic switch to control if the value is ignored when shared/overwritten trough S2S.                                                           |
```
Comment (leventegal-she) [inline docs/SORMAS2SORMAS.md:71]:
```suggestion
| `sormas2sormas.ignoreProperty.additionalDetails` | Controls whether the value in `additionalDetails` field should be sent or not to the other instances. Possible values: true/false(default).|
| `sormas2sormas.ignoreProperty.externalId`        | Controls whether the value in `externalId` field should be sent or not to the other instances. Possible values: true/false(default).|
| `sormas2sormas.ignoreProperty.externalToken`     | Controls whether the value in `externalToken` field should be sent or not to the other instances.  Possible values: true/false(default).|
| `sormas2sormas.ignoreProperty.internalToken`     | Controls whether the value in `internalToken` field should be sent or not to the other instances.  Possible values: true/false(default).|
```

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11696

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11696

**Pull request merged**